### ISSUE TITLE
tou.tv changed layout, tou.pm not working anymore

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#400]
```
What steps will reproduce the problem?
1.trying to download from tou.tv
2.
3.

What is the expected output? What do you see instead?
can't find video ID since they changed site layout


What version of the product are you using? On what operating system?
tried running --update, but was up to date !

Please provide any additional information below.
I tried to switch back to 'pid' rather than 'PID' in the script but no lock 

here's debug : 
~Downloading http://www.tou.tv/perusse-cite/S01E12
-> GET http://www.tou.tv/perusse-cite/S01E12
<- 200 text/html; charset=utf-8 (64158): UTF8 on, non-ASCII, 64158 characters 64
262 bytes
Trying to open plugin C:\Documents and Settings\Chesstu\Application Data/get_fla
sh_videos/plugins/Www.pm
Trying to open plugin C:\Documents and Settings\Chesstu\Application Data/get_fla
sh_videos/plugins/Tou.pm
Using method 'tou' for http://www.tou.tv/perusse-cite/S01E12
Video ID =
Error: Couldn't find TOU.TV video ID at D:\Mes Documents\dowlnload flash video\l
ib/FlashVideo/Site/Tou.pm line 28.

Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

Please confirm the site is using Flash video and if you have Flash available
check that the URL really works(!).

Check for updates by running: get_flash_videos --update

If the latest version does not support this please open a bug (or
contribute a patch!) at http://code.google.com/p/get-flash-videos/
make sure you include the output with --debug enabled.
Couldn't download any videos.




```

Original issue reported on code.google.com by `no1r...@gmail.com` on 28 Jun 2012 at 2:05

**Comment by GoogleCodeExporter** [comment]
```
sorry guys, didn't look in the all issues section.

FIXED as per issue #396 :)

Thanks guys ! 
```

Original comment by `no1r...@gmail.com` on 28 Jun 2012 at 2:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `zakflash...@gmail.com` on 30 Jun 2012 at 7:37
- Changed state: **Invalid**
